tests/Utility/BigIntegerBcmathTest.php

```diff
@@ -12,7 +12,7 @@ class BigIntegerBcmathTest extends BigIntegerTest
 {
     protected function _isSupported()
     {
-        return extension_loaded('bcmath');
+        return true;
```

Comment:
AFAICT this might only be for tests, but wouldn't it try run that suite even when bcmath + the polyfill are unavailable?

Reply:
Right. I now changed both tests to use the same `function_exists` detection as the library for repeatable and meaningful results.